packages/dashboard/src/pages/actions/index.tsx

```diff
@@ -64,7 +64,7 @@ export default function Index() {
 					{actions ? (
 						actions.length > 0 ? (
 							<>
-								<div className={"grid grid-cols-1 gap-6 sm:grid-cols-2"}>
+								<div className={"grid grid-cols-1 gap-6 lg:grid-cols-2"}>
```

Comment:
Action and Template cards are squeezed in medium screens. Changed from `sm` to `lg` to fix it.

Before:

<img width="1477" alt="ResponsivelyApp 2025-04-22 1 06 23 PM" src="https://github.com/user-attachments/assets/2fa92889-7ded-4b30-a015-a42c057afe2e" />

After:

<img width="1475" alt="Opera 2025-04-22 1 05 42 PM" src="https://github.com/user-attachments/assets/545c3122-c725-4dd7-aa4c-11678aa87f13" />